manifests/config.pp

```diff
@@ -21,6 +21,9 @@
 #   Location of symlink that points to configfile. Default: /etc/r10k.yaml
 # * [*forge_settings*]
 #   Hash containing settings for downloading modules from the Puppet Forge.
+# * [*proxy*]
+#   String containing proxy setting for r10k.yaml.
+#   Default: ''
```

Comment:
`params.pp` says it defaults to `undef`.

Reply:
It does not matter. I will create another pull request, which will refactor a lot of stuff and hopefully we will use data-in-modules and generate a REFERENCE.md.